core/server/apps/amp/tests/amp_content_spec.js

```diff
@@ -92,8 +92,7 @@ describe('{{amp_content}} helper', function () {
 
     it('sanitizes remaining and not valid tags', function (done) {
         var testData = {
-                html: '<audio src="https://ia801402.us.archive.org/16/items/EDIS-SRP-0197-06/EDIS-SRP-0197-06.mp3" controls="controls">' +
```

Comment:
I had to remove the `audio` test-data, because Amperize got updated in the meantime and transforms `audio` into `amp-audio` now and therefore, this tag can't be sanitized.